.appveyor.yml

```diff
@@ -0,0 +1,61 @@
+version: 1.{build}
+
+configuration:
+  - Release
+
+clone_folder: c:\projects\miasm
+
+environment:
+  matrix:
+    - platform: Win32
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+      PLATFORM_TOOLSET: v141
+      PYTHON: c:\Python27
+      Z3_INCLUDE_DIRS: c:\projects\triton\build\z3-4.6.0-x86-win\include
+      Z3_LIBRARIES: c:\projects\triton\build\z3-4.6.0-x86-win\bin\libz3.lib
```

Comment:
Yup
I didn't catch this:
Which part is using the environment variable `Z3_include_dirs` & `z3_libraries` ?

Reply:
Shame on me! I've fixed this.